### ISSUE TITLE
compilation error about the select statement located in separate mapper xml with same mapper namespace

### ISSUE THREAD

**Issue by Prachi481992** [mybatis/mybatipse#86]
I'm trying to use a select statement defined in one mapper file in the association in the resultmap of another mapper xml. Both the mapper xmls have the same namespace. I'm using the fully qualified name to refer the select. But the same select is accessible by hyperlink. The code works fine at runtime.
Below is the mapper file which needs to be imported:
```
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.abc.MybatisMapper">
	<!-- add column names only -->
    <select id="getADetails" resultType="com.abc.dto.A">
        SELECT a.id,
         a.name
        FROM
        TABEL_A
        <trim prefix="WHERE" prefixOverrides="AND | OR">
            a.id = #{id}
        </trim>
    </select>
</mapper>
```

Below is the mapper file which uses the select in first mapper file:
```
<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
<mapper namespace="com.abc.MybatisMapper">

    <resultMap id="resultmapB" type="com.abc.dto.B">
       <result property="b_id" column="b_id" />
        <result property="b_name" column="b_name" />
        <association property="a_details" javaType="com.abc.dto.A" column="a_id" select="com.abc.MybatisMapper.getADetails" />
    </resultMap>
<!-- add column names only -->
    <select id="getBDetails" resultMap="resultmapB">
        SELECT b_id,
		b_name,
		a_id
        FROM
        TABEL_B
        <trim prefix="WHERE" prefixOverrides="AND | OR">
            b_id = #{id}
        </trim>
    </select>
</mapper>
```

The line in second file which declares the association gives the compilation error.

**Comment by harawata** [comment]
@Prachi481992 
This should be fixed in 1.1.2 (let me know if it's not).
Thank you for taking the time to report this!

**Comment by Prachi481992** [comment]
@harawata 
The issue is resolved now. 
Thanks for the quick resolution!
